content/rs/security/access-control/manage-users/manage-passwords.md

```diff
@@ -73,7 +73,8 @@ To deactivate password complexity rules:
 
 - Use the `cluster` REST API endpoint to set `password_complexity` to `false`
 
-## Password expiration
+<a href name="enable-password-expiration-policies"></a>
+## Password expiration 
```

Comment:
You can also define anchors with this method.
```suggestion
## Password expiration {#enable-password-expiration-policies}
```

Reply:
@rrelledge - In testing, that syntax only works on the first specified.  We need to use <A> tags to support additional anchors.